### PR TITLE
organization added to activity body

### PR DESCRIPTION
**Pull request by juansebastiancodes**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -301,29 +314,34 @@ module.exports = {
...
       commitId,
-      service ? service : repoName,
+      service ?? repoName,
       environment
     );
     const activityBody = {
       activity: {
         id: commitId,
-        service: service ? service : repoName,
+        service: service ?? repoName,
```
Comment (cesaraugustogarcia) [inline index.js:334]: We cannot support this operators in order to be compatible with Node versions older than 14

Comment (cesaraugustogarcia) [reply index.js:334]: Remember: 
https://github.com/Kelsus/api-spot-package/commit/8b43a9419505a848503206258e85be31e22c3c6b

File: test.js [hunk 2]
```diff
@@ -1,57 +1,109 @@
 const { describe, it } = require("node:test");
 const assert = require("node:assert");
 
-const index = require("./index.js");
-
-describe("Test suite: extractGitHubRepoPath", () => {
```
Comment (cesaraugustogarcia) [inline test.js:6]: Would be nice to have different `describe`s for the two different methods tested: `extractGitHubRepoData` and `buildGitHubUrl`. So in that way we can structure our test suites.

File: index.js [hunk 3]
```diff
@@ -72,35 +73,48 @@ const ACTIVITY_STATUS = "OK";
...
+      if (!name || !server || !owner) continue;
+
+      return { name, server, owner };
+    }
+
+    return null;
+  },
+
+  /**
+   * Format the given url (it can be with ssh or x-access-token format) to classic url like
```
Comment (cesaraugustogarcia) [inline index.js:111]: Good documenting!

File: index.js [hunk 4]
```diff
@@ -301,29 +314,34 @@ module.exports = {
...
+    const {
+      name: repoName,
+      owner: repoOrganization,
+    } = extractGitHubRepoData(repositoryUrl);
 
-    const repoName = module.exports.extractGitHubRepoPath(repositoryUrl);
     const changelog = await module.exports.generateChangelog(
       commitId,
-      service ? service : repoName,
+      service ?? repoName,
```
Comment (cesaraugustogarcia) [inline index.js:328]: We cannot support this operators in order to be compatible with Node versions older than 14

File: index.js [hunk 5]
```diff
@@ -72,35 +73,48 @@ const ACTIVITY_STATUS = "OK";
...
+  },
+
+  /**
+   * Format the given url (it can be with ssh or x-access-token format) to classic url like
+   * 'https://${server}.com/${owner}/${name}'
+   */
+  buildGitHubUrl: (url) => {
+    const { server, name, owner } = module.exports.extractGitHubRepoData(url);
+
+    return `https://${server}.com/${owner}/${name}`;
```
Comment (cesaraugustogarcia) [inline index.js:117]: Probably would be good to return null here as well... in the scenario in which `extractGitHubRepoData` could not parse the given URL, intead of returning a weird URL.

**Review by cesaraugustogarcia — CHANGES_REQUESTED** [review]
- Some changes required
 - Also please add the new supported params to the Readme.
 - Please do not forget to squash before merging.

File: index.js (hunk 1 above)
Comment (juansebastiancodes) [reply index.js:334]: I forgot it !, solving

File: index.js (hunk 3 above)
Comment (juansebastiancodes) [reply index.js:111]: 😁😁

File: index.js (hunk 5 above)
Comment (juansebastiancodes) [reply index.js:117]: Agreed

**Review by cesaraugustogarcia — APPROVED** [review]

**Pull request merged**